monai/networks/layers/filtering.py

```
@@ -221,7 +221,7 @@ def __init__(self, spatial_sigma, color_sigma):
             self.len_spatial_sigma = 3
         else:
             raise ValueError(
-                f"len(spatial_sigma) {spatial_sigma} must match number of spatial dims {self.ken_spatial_sigma}."
+                f"len(spatial_sigma) {spatial_sigma} must match number of spatial dims {self.len_spatial_sigma}."
             )
```

Ruff (0.14.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)